app/farm/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+
+# Adding a stub farmer model for now. This should ideally be linked to the user account created during user
+# registration.
+class FarmerStub(models.Model):
```

Comment:
This should be removed then `makemigrations` again to clean-up the database.

Reply:
done